HLTrigger/Configuration/python/Phase2/ESProducersSources/hltPhase2CSCChannelMapperESSource_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from CalibMuon.CSCCalibration.CSCChannelMapper_cfi import (
+    CSCChannelMapperESSource as _CSCChannelMapperESSource,
+)
+
+hltPhase2CSCChannelMapperESSource = _CSCChannelMapperESSource.clone()
```

Comment:
CMSSW has so far arranged cfis to live in the same package as their code (surely there are exceptions of course) - that eases maintenance and dependencies. Why break that ?

Reply:
Dear @davidlange6 ,

Sorry, I failed I do not understand exactly what is being broken. Is it because we are simply cloning the ESSource with no changes?

There was a bit of discussion on the rationale for doing it like this in #31218 , perhaps that helps clarify our intent!

Reply:
perhaps things are better grouped into cffs indeed if things are just being cloned. But seems a minor issue.

Some things are not - eg Chi2ChargeMeasurementEstimatorESProducer - that sort of cfi is typically put in the same package are its implementation (RecoTracker/MeasurementDet) - but I realize there are just a few of those.

Reply:
I understand! Also, as discussed above, usually HLT configurations live in ConfDB whilst only a dump exists in Github (in HLTrigger/Configuration). Now we are in this transitional state where we want the code to live somewhere official (i.e. in the release) for easy reproducibility, validation, etc. but having it in ConfDB would only add layers of difficulty and not bring any advantages.

In #31218, we started with a more monolithic approach, but we quickly found out that having a 10000-line file that was *not* automatically dumped is no very maintainable. We think that this approach, modular but localized to our own subsystem, strikes a good compromise.